src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -583,20 +681,23 @@ private static Task LogDowngradeWarningsAsync(IEnumerable<RestoreTargetGraph> gr
 
             var projectRestoreCommand = new ProjectRestoreCommand(projectRestoreRequest);
 
-            var result = await projectRestoreCommand.TryRestoreAsync(
-                projectRange,
-                projectFrameworkRuntimePairs,
-                userPackageFolder,
-                fallbackPackageFolders,
-                remoteWalker,
-                context,
-                forceRuntimeGraphCreation: hasSupports,
-                token: token);
+            Tuple<bool, List<RestoreTargetGraph>, RuntimeGraph> result = null;
+            using (var tryRestoreTelemetry = TelemetryActivity.CreateTelemetryActivityWithNewOperationIdAndEvent(telemetryActivity.OperationId, RestoreAttempt))
+            {
+                result = await projectRestoreCommand.TryRestoreAsync(
```

Comment:
where have you change the definition for `TryRestoreAsync` to accept `telemetryActivity` parameter?